config/rbac/role.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
   name: manager-role
```

Comment:
When running `make manifests`, the `role.yaml` file will be changed.

And, it seems that the kubebuilder tags of pytorchjob/tfjob do not take effect.